network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java

```diff
@@ -115,6 +115,49 @@ private <T extends IdentifiableAttributes> Optional<Resource<T>> get(String targ
         return resource;
     }
 
+    private Optional<ExtensionAttributes> getExtensionAttributes(String urlTemplate, Object... uriVariables) {
+        logGetExtensionAttributesUrl(urlTemplate, uriVariables);
+        Stopwatch stopwatch = Stopwatch.createStarted();
+        Optional<ExtensionAttributes> extensionAttributes = restClient.getOneExtensionAttributes(urlTemplate, uriVariables);
+        stopwatch.stop();
+        logGetExtensionAttributesTime(extensionAttributes.isPresent() ? 1 : 0, stopwatch.elapsed(TimeUnit.MILLISECONDS));
+
+        return extensionAttributes;
+    }
+
+    private Map<String, ExtensionAttributes> getExtensionAttributesMap(String urlTemplate, Object... uriVariables) {
+        logGetExtensionAttributesUrl(urlTemplate, uriVariables);
+        Stopwatch stopwatch = Stopwatch.createStarted();
+        Map<String, ExtensionAttributes> extensionAttributes = restClient.get(urlTemplate, new ParameterizedTypeReference<>() { }, uriVariables);
+        stopwatch.stop();
+        logGetExtensionAttributesTime(extensionAttributes.size(), stopwatch.elapsed(TimeUnit.MILLISECONDS));
+
+        return extensionAttributes;
+    }
+
+    private Map<String, Map<String, ExtensionAttributes>> getExtensionAttributesNestedMap(String urlTemplate, Object... uriVariables) {
+        logGetExtensionAttributesUrl(urlTemplate, uriVariables);
+        Stopwatch stopwatch = Stopwatch.createStarted();
+        Map<String, Map<String, ExtensionAttributes>> extensionAttributes = restClient.get(urlTemplate, new ParameterizedTypeReference<>() { }, uriVariables);
+        stopwatch.stop();
+        long loadedAttributesCount = extensionAttributes.values().stream()
+                .mapToLong(innerMap -> innerMap.values().size())
+                .sum();
+        logGetExtensionAttributesTime(loadedAttributesCount, stopwatch.elapsed(TimeUnit.MILLISECONDS));
+
+        return extensionAttributes;
+    }
+
+    private static void logGetExtensionAttributesUrl(String urlTemplate, Object... uriVariables) {
+        if (LOGGER.isInfoEnabled()) {
+            LOGGER.info("Loading extension attributes {}", UriComponentsBuilder.fromUriString(urlTemplate).build(uriVariables));
+        }
+    }
+
+    private static void logGetExtensionAttributesTime(long loadedAttributesCount, long timeElapsed) {
+        LOGGER.info("{} extension attributes loaded in {} ms", loadedAttributesCount, timeElapsed);
```

Comment:
use the same logs as the other cases. Also maybe just use the same code, don't create these functions?

Reply:
ExtensionAttributes are not resources, I'm not sure it makes sense to have the log:
`            LOGGER.info("Loading {} resources {}", target, UriComponentsBuilder.fromUriString(url).buildAndExpand(uriVariables));
` ?
That's why I created these separate functions. If we make get()/getAll() generic as discussed in https://github.com/powsybl/powsybl-network-store/pull/411#discussion_r1701494765, then we can probably avoid creating these functions. ExtensionAttributes can come in a Map so we may need to keep some specific processing here anyway.